src/App.tsx

```diff
@@ -1,47 +1,25 @@
-import React, { useEffect, useState } from 'react';
+import React from 'react';
 import classNames from 'classnames';
 
 import 'bulma/css/bulma.css';
 import '@fortawesome/fontawesome-free/css/all.css';
 import './App.scss';
-
 import { PostsList } from './components/PostsList';
-import { PostDetails } from './components/PostDetails';
 import { UserSelector } from './components/UserSelector';
+import { PostDetails } from './components/PostDetails';
```

Comment:
Ensure that the `PostDetails` component is correctly implemented and exported in the `./components/PostDetails` file. Verify its existence and functionality.